docs/package.json

```diff
@@ -104,5 +105,12 @@
 	"packageManager": "pnpm@8.15.8+sha512.d1a029e1a447ad90bc96cd58b0fad486d2993d531856396f7babf2d83eb1823bb83c5a3d0fc18f675b2d10321d49eb161fece36fe8134aa5823ecd215feed392",
 	"engines": {
 		"node": ">=18.0"
+	},
+	"dependencies": {
+		"@microsoft/applicationinsights-react-js": "^17.3.4",
+		"@microsoft/applicationinsights-web": "^3.3.4",
+		"@wcp/wcp-consent": "^1.1.0",
```

Comment:
I think this needs to be either removed or specified as an `optionalDependency`.